.eslintrc.js

```diff
@@ -0,0 +1,287 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ["plugin:react/recommended", "airbnb"],
```

Comment:
I see that prettier is part of the dependencies, but it's not extended nor put in the plugins. I think you can copy the recommended configuration of the prettier plugin; https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

Reply:
This is actually the first time I am doing this manually, so I was bound to fail lolll

Reply:
will fix when I get to it.

Reply:
No worries, nice that you want to work on this! Maybe you can check out the prettier documentation as well, it has a page on pre-commit hooks; https://prettier.io/docs/en/precommit.html